deepmedic/frontEnd/configParsing/trainSessionParams.py

```diff
@@ -357,42 +466,53 @@ def print_params(self) :
         logPrint("~~Output~~")
         logPrint("Main output folder = " + str(self.mainOutputAbsFolder))
```

Comment:
Print/log the new variable:

```suggestion
        logPrint("Main output folder = " + str(self.mainOutputAbsFolder))
        logPrint("Log performance metrics for tensorboard = " + str(self.tensorboardLog))
```